critiquebrainz/frontend/templates/macros.html

```diff
@@ -51,3 +51,17 @@
     </script>
   {% endif %}
 {% endmacro %}
+
+{% macro spotify_player(spotify_mappings) -%}
+    {% if spotify_mappings %}
+        <iframe src="https://embed.spotify.com/?uri={{ spotify_mappings[0] }}&theme=white"
+                width="100%" height="80" frameborder="0" allowtransparency="true"></iframe>
+        <em><small><a class="text-muted" href="{{ url_for('mapping.spotify_list', release_group_id=id) }}">{{ _('View Spotify mapping information') }}</a></small></em>
+      {% else %}
+        <span class="text-muted">
+          {{ _('We don\'t have a mapping between this release group and Spotify.
+          Please help us find the right album.') }}
+        </span>
+        <div style="text-align:center;font-weight:bold;"><a style="font-size:16px;" href="{{ url_for('mapping.spotify', release_group_id=id) }}" rel="nofollow">{{ _('Match this!') }}</a></div>
```

Comment:
It means that this block offering to add a mapping will now show up on all pages. It might be better to make it optional by having another argument in that macro.